modules/server/src/main/scala/GrpcConfig.scala

```diff
@@ -22,7 +22,7 @@ import java.util.concurrent.Executor
 
 import io.grpc._
 
-sealed trait GrpcConfig                                                        extends Product with Serializable
+trait GrpcConfig                                                               extends Product with Serializable
```

Comment:
Just a question, but what is the reason for unsealing `GrpcConfig`?

Reply:
The `Netty` settings outside this file should extend `GrpcConfig`, to be able to build it with the rest of the "standard" configurations.